deploy/sample-backup-configs/sample_config_build_recipes.yaml

```diff
@@ -11,92 +11,92 @@
 # <NAME>:
 #    DESIGN: <>
 #    TARGET_CONFIG: <>
-#    deploy-triplet: null
+#    deploy_triplet: null
 #    PLATFORM_CONFIG: Config
-#    post-build-hook: null
+#    post_build_hook: null
 #    # REQUIRED: TODO: Will move to "run platform" work
-#    s3-bucket-name: firesim-AWSUSERNAME
+#    s3_bucket_name: firesim-AWSUSERNAME
 
 # Quad-core, Rocket-based recipes
 firesim-rocket-quadcore-nic-l2-llc4mb-ddr3:
```

Comment:
What is the rational for not changing the recipe names?

Reply:
I missed this. The intention was to change the recipe names. I'll go ahead and change this.

Reply:
We should double check that this doesn't cause problems for how metadata is packed into the AGFI description. IIRC "_" might be used as a separator somewhere in there.

Reply:
Good to know, I'll double-check on this (by re-running a buildafi run)

Reply:
I made the assumption checking in the tagging much more robust in https://github.com/firesim/firesim/pull/936 but that hasn't been merged yet.